packages/dialog/src/vaadin-dialog.js

```diff
@@ -26,6 +26,10 @@ registerStyles(
       z-index: 1;
     }
 
+    [part='header'] {
+      flex-wrap: nowrap;
+    }
```

Comment:
Addressing [this comment](https://github.com/vaadin/web-components/pull/4310#issuecomment-1206459013), should we also prevent wrapping the title, and show ellipsis instead?

Alternatively we could try changing the `overflow-wrap` property to `word-break`, so that long words don't overflow.

Reply:
Wrapping the title is generally preferable, as it guarantees that the entire title can be read. Clipping can be achieved with a custom title using header-content slot.